src/Orleans.Hosting.Kubernetes/KubernetesHostingExtensions.cs

```diff
@@ -51,18 +51,7 @@ public static IServiceCollection UseKubernetesHosting(this IServiceCollection se
             services.AddSingleton<ILifecycleParticipant<ISiloLifecycle>, KubernetesClusterAgent>();
 
             // Configure the Kubernetes client.
-            services.AddHttpClient("Orleans.Kubernetes.Agent")
-                .AddTypedClient<IKubernetes>((httpClient, serviceProvider) =>
-                {
-                    var config = serviceProvider.GetRequiredService<KubernetesHostingOptions>().ClientConfiguration;
-                    return new k8s.Kubernetes(
-                        config,
-                        httpClient);
-                }).ConfigurePrimaryHttpMessageHandler(serviceProvider =>
-                {
-                    var config = serviceProvider.GetRequiredService<KubernetesHostingOptions>().ClientConfiguration;
-                    return config.CreateDefaultHttpClientHandler();
-                });
+            //services.AddSingleton<IKubernetes>(_ => new k8s.Kubernetes(KubernetesClientConfig);
```

Comment:
If this is not needed, please delete it and the associated configuration property, `KubernetesHostingOptions.ClientConfiguration`, which should have been consumed as an `IOptions<T>`, but anyway.

Reply:
Exactly the question I wanted to ask you.
In the original code there was a TypedHttpClient added to the services that was never used and I haven't found any other reference of it in the codebase so, not understanding why it was there, I left the comment based on a mock from the KubernetesClient library - seems to me this isn't needed, I'll go ahead and remove it.

Reply:
Sounds good to me